bin/pocs_shell

```diff
@@ -209,32 +209,16 @@ Hardware names: {}   (or all for all hardware)'''.format(
         else:
             print_warning('Please run `setup_pocs` before trying to run')
 
-    def do_collect_status(self, *arg):
-        """Just records the `status` for pocs.
-
-        This would mostly be used when you want to record information from
-        POCS but might be driving the system manually
-
-        """
-        if self.pocs is not None:
-            if self.msg_subscriber is None:
-                self.do_start_messaging()
-
-            print_info("Recording POCS status - Press Ctrl-c to interrupt")
-
-            _keep_going = True
-            while _keep_going:
-                try:
-                    status = self.pocs.status()
-                    pprint(status)
-                    print_info('*' * 80)
-                    self.pocs.db.insert_current('system', status, store_permanently=False)
-                    time.sleep(2)
-                except KeyboardInterrupt:
-                    print_warning('Stopping collection')
-                    _keep_going = False
-        else:
+    def do_status(self, *arg):
```

Comment:
I think the default `do_emptyline` should call this (which is what happens in the mostly unused `DriftShell`)

Reply:
Having the handler for empty line produce a lot of output (or really, to have it do much of anything) seems odd to me.